### DISCUSSION TITLE
Additional Infrastructure Providers for OpenShift

### DISCUSSION THREAD

**Discussion by AdrianHammond** [finos/compliant-financial-infrastructure#295] [Q&A] [answered]
Hi @eddie-knight @abdullahgarcia @thinkl33t @peter-thomas-db 

In https://github.com/finos/ansible-cfi-ocp/issues/3 I am going to add additional infrastructure / cloud providers building on the Google Cloud work done so far. Obvious ones would be Azure and AWS are their other cloud / infra providers I should look at?  Here is a [list](https://docs.openshift.com/container-platform/4.11/installing/index.html#supported-platforms-for-openshift-clusters_ocp-installation-overview) of current infrastructure / cloud providers. 

Thanks A

**Comment by eddie-knight** [comment, marked as answer]
Not sure whether we've ever captured this anywhere, but our philosophy for including work thus far has been (1) Is there anyone ready to create this? and (2) is there anyone ready to consume this? If either are true, it should be captured as a work item. If both are true, it should be prioritized.

I've mentioned it a lot recently, but I think it's really important that we hurry up to refine our process around surveying end users, since that will give us a lot more information about what people are ready to consume. 🤔 Just need to figure out where to track that work & get the ball rolling.

**Reply by AdrianHammond** [reply]
Thanks @eddie-knight  - will close this discussion off and this will be progressed via RIWG team on issue https://github.com/orgs/finos/projects/45/views/1?pane=issue&itemId=13607684
